src/elements/content-sidebar/activity-feed/common/Timestamp.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import Tooltip from '../../../../components/tooltip';
+import ReadableTime from '../../../../components/time/ReadableTime';
+import messages from '../../../common/messages';
+
+type Props = {
+    date: number, // unix epoch timestamp (new Date().getTime())
+};
+
+const Timestamp = ({ date }: Props) => {
+    return (
+        <Tooltip text={<FormattedMessage {...messages.fullDateTime} values={{ time: date }} />}>
+            <small className="bdl-Timestamp">
```

Comment:
Since this is in `/src/elements/content-sidebar`, it should probably use the `bcs-` prefix.